src/transport_ofi.c

```diff
@@ -898,7 +893,7 @@ int populate_av(void)
         if (strncmp(myephostname, ephostname, EPHOSTNAMELEN) == 0) {
             SHMEM_SET_RANK_SAME_NODE(i, num_on_node++);
             if (num_on_node > 255) {
-                RAISE_WARN_STR("ERROR: Too many local ranks");
+                RAISE_WARN_STR("Number of local ranks exceeds limit (255)");
```

Comment:
In other error messages, we have put the return value in the parenthesis. Should we put 255 out of parenthesis here? May be like this:  "Number of local ranks exceeds limit value of 255"